src/com/alevel/module1/Main.java

```diff
@@ -0,0 +1,120 @@
+package com.alevel.module1;
+
+import org.w3c.dom.ls.LSOutput;
+
+import java.util.Random;
+import java.util.Scanner;
+
+
+public class Main {
+    public static void main(String[] args) {
+        final Scanner in = new Scanner(System.in);
+        int[] mass = createArray();
+        System.out.println("Array:");
+        fillRandomArray(mass);
+        outputArray(mass);
+
+        do {
+            try {
+                System.out.print("Enter 1 for sort to max and 2 to sort to min: ");
+                int choose = Integer.parseInt(in.next());
+                if (choose == 1) {
+                    sortToMax(mass);
+                } else if (choose == 2) {
+                    sortToMin(mass);
+                } else {
+                    continue;
+                }
+                break;
+            } catch (Exception ex) {
+                System.out.println("You need to enter int number!");
+            }
+        } while (true);
+        System.out.println("Sorted array:");
+        outputArray(mass);
+
+        System.out.println("Array with degree:");
+        powArray(mass);
+        outputArray(mass);
+
+        System.out.println("Lock array:");
+        Lock lock = new Lock(mass);
+        lock.getArray()[0] = 2;
+        lock.printArray();
+    }
+
+    static int[] createArray() {
+        final Scanner in = new Scanner(System.in);
+        do {
+            try {
+                System.out.print("Enter the number of elements in the array from 10: ");
+                int length = Integer.parseInt(in.next());
```

Comment:
You can use in.nextInt()

Reply:
In this case, if the input is incorrect, the flow is clogged. This does not happen with the next method.

Reply:
You can use in.hasNextInt()